docs/reference/contactproperties.md

```diff
@@ -73,8 +73,7 @@ Solid objects with velocities below this threshold will have a `bounce` value se
 Consider a contact point where the coefficient of friction mu is infinite.
 Normally, if a force f is applied to the two contacting surfaces, to try and get them to slide past each other, they will not move.
 However, if the FDS coefficient is set to a positive value k then the surfaces will slide past each other, building up to a steady velocity of k*f relative to each other.
-Note that this is quite different from normal frictional effects: the force does not cause a constant acceleration of the surfaces relative to each other
-- it causes a brief acceleration to achieve the steady velocity."
+Note that this is quite different from normal frictional effects: the force does not cause a constant acceleration of the surfaces relative to each other \- it causes a brief acceleration to achieve the steady velocity."
```

Comment:
Shouldn't use a mdash: &mdash; here?

Reply:
(or a ndash: &ndash; ?)

Reply:
I would use the mdash.

Note that ODE manual really uses a hypen (even if it is "wrong").